Sources/Commands/SwiftTestTool.swift

```diff
@@ -167,11 +167,12 @@ public struct SwiftTestTool {
         //     .........10.........20.........30.........40.........50.........60.........70..
         print("OVERVIEW: Build and run tests")
         print("")
-        print("USAGE: swift test [specifier] [options]")
+        print("USAGE: swift test [mode] [options]")
         print("")
-        print("SPECIFIER:")
+        print("MODES:")
```

Comment:
This should probably be `OPTIONS` not `MODES` (and same for line above), since `-s` is an option... its better to just use the generic term IMHO.